components/automate-gateway/handler/iam/v2/introspect/introspect.go

```diff
@@ -129,22 +129,27 @@ func (a *AuthzServer) Introspect(
 		return nil, err
 	}
 
-	// Replace endpoint with realized path
-	// Note: For POST /introspect (this handler), it should be safe to assume
-	// that there's only ever one endpoint under consideration.
```

Comment:
... Unless you use different variable names in parameterized endpoints in the proto file! 🤷